changelogs/fragments/360-bgp-connection-afi.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_modify - field name change in ``routing bgp connection`` path implemented by RouterOS 7.19 and newer (https://github.com/ansible-collections/community.routeros/pull/360).
```

Ah sorry, there's one detail - this also applies to the `api_info` module:
```suggestion
  - api_modify, api_info - field name change in ``routing bgp connection`` path implemented by RouterOS 7.19 and newer (https://github.com/ansible-collections/community.routeros/pull/360).
```

---

Sharp observation! (especially in the early morning ☕)